quickstarts/minikube/index.md

```diff
@@ -0,0 +1,44 @@
+# Starting Minikube 
+
+This assumes that you have the latest version of the `minikube` binary, which you can get [here](https://kubernetes.io/docs/setup/minikube/#installation).
+
+```shell
+minikube start --memory=8192 --cpus=4 \
+  --kubernetes-version=v1.12.1 \
+  --vm-driver=kvm2 \
+  --bootstrapper=kubeadm \
+  --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
```

Comment:
what's wrong with my `minikube` tries where I use just `start` and `--vm-driver` because of kvm? Isn't it too much stuff for a quick start?